assets/downloads/tutorials/extended-game-java/Drop.java

```diff
@@ -1,18 +1,29 @@
 package com.badlogic.drop;
 
 import com.badlogic.gdx.Game;
+import com.badlogic.gdx.Gdx;
 import com.badlogic.gdx.graphics.g2d.BitmapFont;
 import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+import com.badlogic.gdx.utils.viewport.FitViewport;
+
 
 public class Drop extends Game {
 
-	SpriteBatch batch;
-	BitmapFont font;
+	public SpriteBatch batch;
+	public BitmapFont font;
+	public FitViewport viewport;
+
 
 	public void create() {
 		batch = new SpriteBatch();
-		// Use LibGDX's default Arial font.
+		// use libGDX's default Arial font
 		font = new BitmapFont();
+		viewport = new FitViewport(8, 5);
+
+		//font has 15pt, but we need to scale it to our viewport by ratio of viewport height to screen height
+		font.setUseIntegerPositions(false);
+		font.getData().setScale(viewport.getWorldHeight() / Gdx.graphics.getHeight());
```

Comment:
Previously camera was set as 1px = 1m. I changed it as it's now in updated version of Simple game (100px = 1m)  but it requires font scaling and this was my only idea how to do that